base/deprecated.jl

```diff
@@ -259,3 +257,9 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool!             rand!
+@deprecate randbool()            rand(Bool)
+@deprecate randbool(dims)        bitrand(dims)
+@deprecate randbool(dims...)     bitrand(dims)
+@deprecate randbool(r, dims)     bitrand(r, dims)
+@deprecate randbool(r, dims...)  bitrand(r, dims)
```

Comment:
`randbool(1, 2, 3)` won't work, and `randbool(MersenneTwister())` won't work as expected, I think you can't avoid using type annotations here!